gtk4/sys/build.rs

```diff
@@ -11,7 +15,37 @@ fn main() {} // prevent linking libraries to avoid documentation failure
 #[cfg(not(feature = "dox"))]
 fn main() {
     if let Err(s) = system_deps::Config::new().probe() {
-        println!("cargo:warning={}", s);
+        let _ = writeln!(io::stderr(), "{}", s);
         process::exit(1);
     }
+
+    // It's safe to assume we can call this because we found the library OK
+    // in find()
+    check_features();
+}
+
+#[cfg(not(feature = "dox"))]
+fn check_features() {
```

Comment:
It would be best to do this in the `gdk4-sys` crate. Why it's the same pkg-config file, the backend is relevant to gdk and not to gtk.